README.md

```diff
@@ -142,5 +167,17 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
```

Comment:
I'm gonna assume `person` is a post-like object that you defined up here somewhere 👍 😉